server/src/internalClusterTest/java/org/opensearch/search/SearchCancellationIT.java

```diff
@@ -86,12 +91,29 @@
 import static org.hamcrest.Matchers.notNullValue;
 
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.SUITE)
-public class SearchCancellationIT extends OpenSearchIntegTestCase {
+public class SearchCancellationIT extends ParameterizedOpenSearchIntegTestCase {
 
     private TimeValue requestCancellationTimeout = TimeValue.timeValueSeconds(1);
     private TimeValue clusterCancellationTimeout = TimeValue.timeValueMillis(1500);
     private TimeValue keepAlive = TimeValue.timeValueSeconds(5);
 
+    public SearchCancellationIT(Settings settings) {
+        super(settings);
+    }
+
+    @ParametersFactory
+    public static Collection<Object[]> parameters() {
+        return Arrays.asList(
+            new Object[] { Settings.builder().put(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey(), false).build() },
+            new Object[] { Settings.builder().put(CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING.getKey(), true).build() }
+        );
+    }
+
+    @Override
+    protected Settings featureFlagSettings() {
```

Comment:
Referring to @sohami comment (https://github.com/opensearch-project/OpenSearch/pull/9083#discussion_r1287731890), featureFlagSettings() and nodeSettings() would have no effect on `OpenSearchIntegTestCase.Scope.SUITE` right now, it that correct?

Reply:
yes thats correct, this mechanism would work for both `OpenSearchIntegTestCase.Scope.SUITE` and `OpenSearchIntegTestCase.Scope.TEST`